variables.tf

```diff
@@ -10,13 +10,33 @@ variable "rg_connectivity_name" {
   description = "Resource Group name where Azure Bastion Subnet is located"
   type        = string
 }
-variable "rg_management_name" {
-  description = "Resource Group name where Azure Bastion Service will be deployed to"
+# variable "rg_management_name" {

```

Comment:
All code is preserved by git. Instead of commenting it out I would just delete it to avoid confusion